Packs/FeedCognyteLuminar/Integrations/CognyteLuminar/CognyteLuminar.py

```diff
@@ -0,0 +1,463 @@
+            'lastseenbysource': parent.get('created'),
+            "shortdescription": parent.get('description') or ""
+        }
+
+        indicator = {
+            'value': children.get("account_login"),
+            'occurred': datetime.strptime(parent.get("created"),
+                                          '%Y-%m-%dT%H:%M:%S.%f%z').strftime(
+                "%m/%d/%Y, %H:%M:%S"),
+            'type': FeedIndicatorType.Account,
+            'rawJSON': dict(children),
+            'fields': additional_field,
+        }
+        modified_childrens.append(indicator)
+    return parent, modified_childrens
+
+
+def enrich_malware_items(parent, childrens):
+    """
+    This will get Malware, Indicator objects
+    Args:
+        parent: Malware
+        childrens: Indicator
+
+    Returns: Malware, Indicator Objects
+
+    """
+    modified_childrens = []
+    modified_parent = None
+    all_tags = set()
+    for children in childrens:
+        pattern = children.get("pattern")
+        for match in stix_regex_parser.findall(pattern):
+            stix_type, _, value = match
+            malware_types = parent.get("malwareTypes")
+            if stix_type in LUMINAR_TO_XSOAR_TYPES.keys():
+                indicator_type = LUMINAR_TO_XSOAR_TYPES[stix_type]
+            else:
+                indicator_type = stix_type
+            children.update(indicator_type=indicator_type)
+            parent["name"] = children["name"]
+            children["value"] = value
+            relationships_lst = EntityRelationship(
+                name=EntityRelationship.Relationships.INDICATED_BY,
+                entity_a=value,
+                entity_a_type=indicator_type,
+                entity_b=parent["name"],
+                entity_b_type="STIX Malware",
+            )
+            tags = [it for it in children["indicator_types"]]
+            all_tags.update(tags)
+            tags.append("Malware Family: " + parent["name"])
+            malware_types_str = ",".join(malware_types)
+            indicator_type_str = ",".join(children["indicator_types"])
+            tags.append(f"Malware Type: {malware_types_str}")
+
+            indicator = {
+                'value': value,
+                'occurred': datetime.strptime(children["created"],
+                                              '%Y-%m-%dT%H:%M:%S.%f%z').strftime(
+                    "%m/%d/%Y, %H:%M:%S"),
+                'type': indicator_type,
+                'rawJSON': dict(children),
+                'fields': {
+                    "category": "Malware",
+                    "malwarefamily": children["name"],
+                    "shortdescription": f"This {indicator_type} was involved in {indicator_type_str}",
+                    "creationdate": children["created"],
+                    'firstseenbysource': children["created"],
+                    "updateddate": children["modified"],
+                    'lastseenbysource': children["modified"],
+                    "tags": tags
```

Comment:
what about the tags from the integration configuration? doesn't seem like they are handled